### DISCUSSION TITLE
Let's Encrypt cert revocation this week?

### DISCUSSION THREAD

**Discussion by kamranayub** [ravendb/ravendb#13561] [Q&A] [answered]
Just wondering if RavenDB LE certs are impacted by this upcoming change:

https://www.bleepingcomputer.com/news/security/lets-encrypt-is-revoking-lots-of-ssl-certificates-in-two-days/

**Comment by gregolsky** [comment, marked as answer]
No, RavenDB LE certs are not impacted. We use DNS challenge for our certificates, not `TLS-ALPN-01`.
